init.lua

```diff
@@ -262,7 +262,12 @@ if legacy_setting_getbool("item_drop.enable_item_pickup",
 
 		local pos = player:get_pos()
 		pos.y = pos.y+0.5
-		local inv
+		local inv = player:get_inventory()
+		if not inv then
+			minetest.log("error", "[item_drop] Couldn't " ..
+				"get inventory")
+			return
+		end
```

Comment:
According to lua_api, inv is never nil for players, so this check can be removed or replaced with an assertion.
```suggestion
```